tests/v2/validation/provisioning/config.go

```diff
@@ -45,13 +46,14 @@ func (c TestClientName) String() string {
 }
 
 type Config struct {
-	NodesAndRoles          []machinepools.NodeRoles `json:"nodesAndRoles" yaml:"nodesAndRoles" default:"[]"`
-	NodesAndRolesRKE1      []nodepools.NodeRoles    `json:"nodesAndRolesRKE1" yaml:"nodesAndRolesRKE1" default:"[]"`
-	K3SKubernetesVersions  []string                 `json:"k3sKubernetesVersion" yaml:"k3sKubernetesVersion"`
-	RKE1KubernetesVersions []string                 `json:"rke1KubernetesVersion" yaml:"rke1KubernetesVersion"`
-	RKE2KubernetesVersions []string                 `json:"rke2KubernetesVersion" yaml:"rke2KubernetesVersion"`
-	CNIs                   []string                 `json:"cni" yaml:"cni"`
-	Providers              []string                 `json:"providers" yaml:"providers"`
-	NodeProviders          []string                 `json:"nodeProviders" yaml:"nodeProviders"`
-	Hardened               bool                     `json:"hardened" yaml:"hardened"`
+	NodesAndRoles            []machinepools.NodeRoles       `json:"nodesAndRoles" yaml:"nodesAndRoles" default:"[]"`
+	NodesAndRolesRKE1        []nodepools.NodeRoles          `json:"nodesAndRolesRKE1" yaml:"nodesAndRolesRKE1" default:"[]"`
+	K3SKubernetesVersions    []string                       `json:"k3sKubernetesVersion" yaml:"k3sKubernetesVersion"`
+	RKE1KubernetesVersions   []string                       `json:"rke1KubernetesVersion" yaml:"rke1KubernetesVersion"`
+	RKE2KubernetesVersions   []string                       `json:"rke2KubernetesVersion" yaml:"rke2KubernetesVersion"`
+	LocalClusterAuthEndpoint rkev1.LocalClusterAuthEndpoint `json:"localClusterAuthEndpoint" yaml:"localClusterAuthEndpoint"`
+	CNIs                     []string                       `json:"cni" yaml:"cni"`
+	Providers                []string                       `json:"providers" yaml:"providers"`
+	NodeProviders            []string                       `json:"nodeProviders" yaml:"nodeProviders"`
+	Hardened                 bool                           `json:"hardened" yaml:"hardened"`
```

Comment:
So just to be safe on this config change, it might be safer to rebase this again since the branch is 112 commits behind the current head